js_modules/dagster-ui/packages/ui-core/src/partitions/OrdinalPartitionSelector.tsx

```diff
@@ -0,0 +1,180 @@
+import {
+  Box,
+  Checkbox,
+  Colors,
+  Icon,
+  Menu,
+  MenuDivider,
+  MenuItem,
+  MiddleTruncate,
+  TagSelectorDropdownItemProps,
+  TagSelectorDropdownProps,
+  TagSelectorWithSearch,
+} from '@dagster-io/ui-components';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import {PartitionStatusHealthSource} from './PartitionStatus';
+import {AssetPartitionStatusDot} from '../assets/AssetPartitionList';
+import {partitionStatusAtIndex} from '../assets/usePartitionHealthData';
+import {RunStatus} from '../graphql/types';
+import {RunStatusDot} from '../runs/RunStatusDots';
+
+export const OrdinalPartitionSelector = ({
+  allPartitions,
+  selectedPartitions,
+  setSelectedPartitions,
+  setShowCreatePartition,
+  isDynamic,
+  health,
+  placeholder,
+  mode = 'multiple',
+}: {
+  allPartitions: string[];
+  selectedPartitions: string[];
+  setSelectedPartitions: (tags: string[]) => void;
+  health: PartitionStatusHealthSource;
+  setShowCreatePartition?: (show: boolean) => void;
+  isDynamic: boolean;
+  placeholder?: string;
+  mode?: 'single' | 'multiple';
```

Comment:
The only changes here are the addition of these two props -- just moved it to it's own file